phpBB/includes/mcp/mcp_front.php

```diff
@@ -119,11 +119,32 @@ function mcp_front_view($id, $mode, $action)
 						AND t.topic_id = p.topic_id
 						AND p.poster_id = u.user_id
 					ORDER BY p.post_time DESC, p.post_id DESC';
+
+				/**
+				* Alter posts data SQL query
```

Comment:
This doc-block does not follow the same (and suggested IMO) indentation of the others. See `*`'s